server/src/main/java/io/pravega/schemaregistry/server/rest/resources/GroupResourceImpl.java

```diff
@@ -0,0 +1,690 @@
+                            getGroupResource(x.getKey()) :
+                            getGroupResource(x.getKey(), namespace);
+
+                    authenticateAuthorize(authorizationHeader, resource, READ);
+                    return true;
+                } catch (AuthException e) {
+                    return false;
+                }
+            };
+            
+            return FuturesCollector.filteredWithTokenAndLimit(
+                    (ContinuationToken t, Integer l) ->
+                            getRegistryService().listGroups(namespace, t, l)
+                                           .thenApply(mwt -> new AbstractMap.SimpleEntry<>(mwt.getToken(),
+                                                   new ArrayList<>(mwt.getMap().entrySet()))), 
+                    predicate, ContinuationToken.fromString(continuationToken), toFetch, getExecutorService())
+                                   .thenAccept(result -> {
+                                          String contToken = result.getKey() == null ?
+                                                  ContinuationToken.EMPTY.toString() : result.getKey().toString();
+                                          groupsList.groups(
+                                                  result.getValue().stream().collect(
+                                                          Collectors.toMap(Map.Entry::getKey, x -> ModelHelper.encode(x.getValue()))))
+                                                    .setContinuationToken(contToken);
+                                      })
+                                   .thenApply(r -> Response.status(Status.OK).entity(groupsList).build())
+                                   .exceptionally(exception -> {
+                       log.warn("listGroups failed with exception: ", Exceptions.unwrap(exception));
+                       return Response.status(Status.INTERNAL_SERVER_ERROR).build();
+                   })
+                                   .thenApply(response -> {
+                       asyncResponse.resume(response);
+                       return response;
+                   });
+        });
+    }
+
+    @Override
+    public void createGroup(CreateGroupRequest createGroupRequest, String namespace,
+                            AsyncResponse asyncResponse) {
+        String resource = Strings.isNullOrEmpty(namespace) ? getNamespaceResource() : getNamespaceResource(namespace);
+        withCompletion("createGroup", READ_UPDATE, resource, asyncResponse, () -> {
+            GroupProperties groupProperties = ModelHelper.decode(createGroupRequest.getGroupProperties());
+            String group = createGroupRequest.getGroupName();
+            return getRegistryService().createGroup(namespace, group, groupProperties)
+                                  .thenApply(createStatus -> {
+                                      if (!createStatus) {
+                                          log.info("group {} exists", group);
```

Comment:
`debug` at best

Reply:
No, its better at info. We log each request and its response to be sent at INFO. 
So we know what exactly happened to the processing.